app/config/sources.inc.php

```diff
@@ -316,8 +321,10 @@
         'supported_locales' => $mozillaorg,
     ],
     'firefox/sendto.lang' => [
-        'priority'          => 1,
+        'flags'             => [
+          'obsolete'        => ['all'],
```

Comment:
The indentation is wrong, but more important, this change is **really** wrong. We're adding a page that describe the "Send to" feature, this file is the widget to send SMS on mozilla.org…